File: laravel-app/app/Models/PatientProfile.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class PatientProfile extends Model
{
    protected $fillable = [
        'user_id',
        'phone',
        'dob',
        'gender',
        'insurance_number',
        'citizen_id',
        'address',
        'emergency_contact_name',
        'emergency_contact_phone',
        'medical_history',
        'allergies',
    ];

    protected function casts(): array
    {
        return [
            'dob' => 'date',
        ];
    }

    public function user(): BelongsTo
    {
        return $this->belongsTo(User::class);
    }
}

File: android-patient-app/app/src/main/java/com/huy/smartqueue/data/model/PatientProfile.kt
package com.huy.smartqueue.data.model

import com.google.gson.annotations.SerializedName

data class PatientProfile(
    val phone: String,
    val dob: String?,
    val gender: String?,
    @SerializedName("insurance_number")
    val insuranceNumber: String?,
    @SerializedName("citizen_id")
    val citizenId: String?,
    val address: String?,
    @SerializedName("emergency_contact_name")
    val emergencyContactName: String?,
    @SerializedName("emergency_contact_phone")
    val emergencyContactPhone: String?,
    @SerializedName("medical_history")
    val medicalHistory: String?,
    val allergies: String?
)
